content/docs/getting_started/quickstart/manual_demo.md

```diff
@@ -146,7 +145,7 @@ kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/release-v
 Create the `bookstore` service account, service, and deployment:
 
 ```bash
-kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/release-v0.9/docs/example/manifests/apps/bookstore.yaml
+kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/release-v0.9/docs/example/manifests/apps/bookstore-v1.yaml
```

Comment:
Let's update the refs to point to main for the main branch version of the docs. We can document in the release guide to replace `main` with `release-xyz`.